Gemfile.lock

```diff
@@ -539,7 +539,7 @@ DEPENDENCIES
   will_paginate-bootstrap (>= 1.0.1)
 
 RUBY VERSION
-   ruby 2.4.1p111
```

Comment:
This file need to be removed too

Reply:
Could u please elaborate regarding wat u r referring in gemfile...?

Reply:
Whole gemfile is not needed